src/auth/negotiate/kerberos/Makefile.am

```diff
@@ -10,7 +10,8 @@ include $(top_srcdir)/src/Common.am
 EXTRA_DIST= \
 	README \
 	required.m4 \
-	negotiate_kerberos_auth.8
+	negotiate_kerberos_auth.8 \
+	negotiate_external_acl
```

Comment:
external ACL helpers go under src/acl/external/FOO/ now with ext_FOO_acl naming.

Reply:
OK It was intended as a simple example to show how the Kerberos PAC field extracted by the auth helper can be efficiently used.  If you agree I can move it to src/acl/external/negotiate_kerberos_group